packages/web-app-files/src/views/FilesDrop.vue

```diff
@@ -10,7 +10,7 @@
       </div>
       <div v-else key="loaded-drop" class="uk-flex uk-flex-column uk-flex-middle uk-height-1-1">
         <div class="uk-text-center uk-width-1-1 uk-width-xxlarge@m">
-          <h2 v-text="title" />
+          <h2 data-testid="files-drop-share-owner-header" v-text="title" />
```

Comment:
is it okay to match the whole wrapper snapshot instead of pointing to a specific element?? e.g. cases like
```
it('should show page title', () => {
  // setup
  expect(wrapper)..toMatchSnapshot()
}
it('should show loading text', () => {
  // setup
  expect(wrapper)..toMatchSnapshot()
}
```

Reply:
I don't fully understand the question I fear, but yeah you can/should always make use of the whole wrapper snapshot:)

Reply:
that actually answers my question. I was just confused whether to use the whole wrapper for the snapshot or we'd just grab the particular element that we'd want for assertions.